src/data_hub/lasso_callback.js

```diff
@@ -0,0 +1,32 @@
+// callback on lasso selection
+
+function debounce(func, wait) {
+    let timeout;
+    return function (...args) {
+        const context = this;
+        clearTimeout(timeout);
+        timeout = setTimeout(() => func.apply(context, args), wait);
+    };
+}
+
```

Comment:
@kennyjwilli does this make sense? i don't want to hit the API a million times a minute

Reply:
Probably, yes. We have lodash available, and you should be able use `_.throttle` ([docs](https://lodash.com/docs/4.17.15#throttle)) if you'd like.